src/Answer.jsx

```diff
@@ -1,22 +1,27 @@
-import React, { Component } from 'react';
-import { Link } from 'react-router-dom';
-
-class Answer extends Component {
-  render() {
-    return (
-      <div>
-        <p> {this.props.data.question} </p>
-        <p> {this.props.data.answer} </p>
-        <Link to={'/'}>
-          <button className="back-button">Назад</button>
-        </Link>
-      </div>
-    );
-  }
-}
-
-Answer.propTypes = {
-  data: React.PropTypes.object.isRequired,
-};
-
-export default Answer;
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
```

Comment:
coz this is new feature of React@15.5 
https://facebook.github.io/react/blog/2017/04/07/react-v15.5.0.html#migrating-from-react.proptypes